app/models/podcast.rb

```diff
@@ -129,13 +129,7 @@ def managing_editor
   end
 
   def feed_episodes
-    feed = []
-    feed_max = display_episodes_count.to_i
-    episodes.published.each do |ep|
-      feed << ep if ep.include_in_feed?
-      break if (feed_max > 0) && (feed.size >= feed_max)
-    end
-    feed
+    default_feed.feed_episodes
```

Comment:
Makes more sense to have this at the feed level - delegating to the feed for compatibility